debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -70,6 +74,7 @@
      */
     public SqlServerConnection(Configuration config) {
         super(config, FACTORY);
+        lsnToInstantCache = new BoundedConcurrentHashMap<>(100);
```

Comment:
Is it ok to have a hardcoded cache size?

Reply:
Yes, I think that's fine.